sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationManager.kt

```diff
@@ -155,6 +158,48 @@ interface ViewAnnotationManager {
    */
   fun getViewAnnotationUpdateMode(): ViewAnnotationUpdateMode
 
+  /**
+   * Sets the visible region so that the map displays the specified annotations.
+   *
+   * This method takes advantage of [ViewPort] plugin and uses [OverviewViewportState] to set the geometry
+   * of specified annotations and apply [ViewportTransition] to move camera to target region.
+   *
+   * @param annotations list of all the views that needs to fit in the visible frame.
+   * @param edgeInsets paddings to apply.
+   * @param bearing camera bearing to apply.
+   * @param pitch camera pitch to apply.
+   * @param animationDurationMs animation duration in millis, default to [DEFAULT_ANNOTATION_ANIMATION_DURATION]
+   *
+   * This API is not supported by Globe projection.
+   */
+  fun showAnnotations(
```

Comment:
What does this method actually change? Only visible region / camera? IF so - to me the name seems a bit misleading, I'd expect it actually adds shows (adds) the annotations

Reply:
it basically update camera so that all annotations provided will fit in the visible region. 

> to me the name seems a bit misleading, I'd expect it actually adds shows (adds) the annotations

The name is kept similar to [ios v6](https://docs.mapbox.com/ios/maps/api/6.4.1/Classes/MGLMapView.html#/c:objc(cs)MGLMapView(im)showAnnotations:animated:). I guess we can improve docs a bit to be clear about the api usage.

Reply:
we're using default args in those methods - do we need to mark them as `@JvmOverloads`?

Reply:
@ank27 @maios - after some offline discussion with the team I realized that `showAnnotations` indeed looks redundant. It may have been useful in v6 / v9 where we did not have flexible camera animation system but now it looks strange that we introduce new method `showAnnotations` (naming is confusing as users could think that this utility method makes given view annotations View.VISIBLE from View.INVISIBLE) that allows to create the camera `flyTo` with one parameter - duration. What if the user want to change let's say some other `MapAnimationOptions` like `owner`, `startDelay` etc? 

My proposal is to remove this method at all and leave the only `cameraForAnnotations` returning `CameraOptions` and point user to the animation plugin documentation allowing to customize that animation in any way.

Reply:
> it basically update camera so that all annotations provided will fit in the visible region.

right, and in Android world it's common to have `.show***` methods that actually change the views visibility - that's why it seems mesleading to me. 

I also think that methods to update the camera would be better placed in the Viewport plugin or MapboxMap itself.

Reply:
> My proposal is to remove this method at all and leave the only cameraForAnnotations returning CameraOptions and point user to the animation plugin documentation allowing to customize that animation in any way.

Correct, I am too in favor of just keeping `cameraForAnnotations`, and letting users create their own animations. they can always use either `mapboxMap.flyTo` or `viewport.transitionTo` , which allows them to control and customize the animation options.

Reply:
agree with above assessment, in the view annotation, I think we just need to expose the cameraForAnnotations. We can build an example of a custom ViewportState to work with viewport plugin, or build a ViewportState separately inside the viewport plugin.